experimental/ModelBuilder/ModelRunner.h

```diff
@@ -85,11 +85,14 @@ class ModelRunner {
   // Compile the owned `module` into LLVMIR that can be passed to the buffer.
   // For now, the MLIR passes and transformations are kept to a minimum and only
   // perform straightforward lowering to LLVMIR.
-  // An optional CompilationOptions object is passed to control special passes
+  // An optional CompilationOptions object is passed to control special passes.
+  // An optional array of extra symbols can be given.
   // An optional array of shared runtime support libraries is passed to the
   // execution engine.
-  void compile(CompilationOptions compilationOptions,
-               llvm::ArrayRef<const std::string> runtime = None);
+  void compile(
+      CompilationOptions compilationOptions,
+      llvm::ArrayRef<std::pair<std::string, void *>> extra_symbols = None,
```

Comment:
Can you make it the last argument? Otherwise it will break some of our existing ModelBuilder tests. 
An alternative solution is to update the tests passing runtime libraries. 

Right now the experimental folder is not built by default. You can add the CMake option -DIREE_BUILD_EXPERIMENTAL=ON and do cmake --build to make sure model builder tests are still compiling correctly.

Reply:
Thanks for the comment!

I think I fixed the concern. First time submitting a pull request, so lemme know if it's still borked :)